src/screens/addContact/AddContact.js

```diff
@@ -0,0 +1,63 @@
+import React, { useState } from 'react'
+import { View, Text, TextInput, TouchableOpacity } from 'react-native'
+
+const AddContact = () => {
+  const [contact, setContact] = useState({
+    name: '',
+    phoneNumber: '',
+    email: '',
+    dob: ''
+  })
+
+  const handleInputChange = (field, value) => {
+    setContact({ ...contact, [field]: value })
+  }
+
+  const handleAddContact = () => {
+    // Handle adding the contact to the database or storage here
+    console.log('Add Contact pressed')
+  }
+
+  return (
+    <View>
+      <Text>Add Contact</Text>
+      <TextInput
+        style={{ height: 40, borderColor: 'gray', borderWidth: 1 }}
+        placeholder="Name"
+        onChangeText={text => handleInputChange('name', text)}
+        defaultValue={contact.name}
+      />
+      <TextInput
+        style={{ height: 40, borderColor: 'gray', borderWidth: 1 }}
+        placeholder="Phone Number"
+        onChangeText={text => handleInputChange('phoneNumber', text)}
+        defaultValue={contact.phoneNumber}
+      />
```

Comment:
Looks like we are repeating ourselves, so it's better to create a `inputField` component.